addons/web_tour/static/src/tour_service/tour_service.js

```diff
@@ -32,25 +32,31 @@ import { callWithUnloadCheck } from "./tour_utils";
  *
  * @typedef TourStep
  * @property {string} [id]
- * @property {JQuerySelector} trigger
- * @property {JQuerySelector} [extra_trigger]
- * @property {JQuerySelector} [alt_trigger]
- * @property {JQuerySelector} [skip_trigger]
- * @property {string} [content]
- * @property {"top" | "botton" | "left" | "right"} [position]
+ * @property {HootSelector} trigger The node on which the action will be executed.
+ * @property {HootSelector} [extra_trigger] Required (extra) node for the step to be executed.
+ * @property {HootSelector} [alt_trigger] An alternative node to the trigger (trigger or alt_trigger).
+ * @property {HootSelector} [skip_trigger] If node is present in the DOM, it bypasses the step.
+ * @property {string} [content] Description of the step.
+ * @property {"top" | "botton" | "left" | "right"} [position] The position where the UI helper is shown.
  * @property {"community" | "enterprise"} [edition]
- * @property {RunCommand} [run]
+ * @property {RunCommand} [run] Run a function wether triggers conditions are verified.
+ * @property {boolean} [allowInvisible] Allow that trigger nodes (ALL) can be invisibles.
+ * @property {boolean} [allowDisabled] Allow that trigger node (ONLY) can be disabled.
```

Comment:
all and only ?? What do you mean?

Reply:
AllowInvisible for all triggers => (trigger, alt_trigger, skip_trigger, extra_trigger) 
AllowDisabled only for trigger => (trigger)

Reply:
Then I would write:

* @property {boolean} [allowInvisible] Allow trigger nodes (any of them) to be invisible
* @property {boolean} [allowDisabled] Allow the trigger node to be disabled.